### ISSUE TITLE
Testing broken in VS Code (Inappropriate ioctl for device)

### ISSUE THREAD

**Issue by EricWardner** [Gallopsled/pwntools#1407]
I've been playing around with VS Code and the Windows Subsystem for Linux trying to get some of my projects working with that environment. 

I'm having an issue that I think might be related to the VS Code output terminal. Here's an example that breaks when I try to run tests from from inside the IDE's built in testing framework but not when I run from the IDE command line. I imagine it has something to do with the way VS code imports the module into it's own testing framework. 

## Code That Breaks 
```python
import unittest
from pwn import *

class TestExercise(unittest.TestCase):
    def test_remote(self):
        conn = remote('ftp.ubuntu.com',21)
        resp = conn.recvline().decode()
        print(resp)
        self.assertIn("220", resp)
    
if __name__ == '__main__':
    unittest.main()
```

## Thrown Exception

```
ImportError: Failed to import test module: runtests
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "*****/projects/random_pwntools_test/runtests.py", line 2, in <module>
    from pwn import *
  File "*****/.local/lib/python3.6/site-packages/pwn/__init__.py", line 6, in <module>
    pwnlib.args.initialize()
  File "*****/.local/lib/python3.6/site-packages/pwnlib/args.py", line 209, in initialize
    term.init()
  File "*****/.local/lib/python3.6/site-packages/pwnlib/term/__init__.py", line 75, in init
    term.init()
  File "*****/.local/lib/python3.6/site-packages/pwnlib/term/term.py", line 102, in init
    setupterm()
  File "*****/.local/lib/python3.6/site-packages/pwnlib/term/term.py", line 74, in setupterm
    update_geometry()
  File "*****/.local/lib/python3.6/site-packages/pwnlib/term/term.py", line 46, in update_geometry
    hw = fcntl.ioctl(fd.fileno(), termios.TIOCGWINSZ, '1234')
OSError: [Errno 25] Inappropriate ioctl for device
```

### Hacky Fixes
Just shimming out the initialize function works for my specific use case. But that probably kills other terminal functionality I might want in the future? (and it makes command line output way less pretty)
```python
import pwnlib.args
pwnlib.args.initialize = lambda: None
```

### Resources
https://code.visualstudio.com/docs/python/testing#_test-discovery

**Comment by EricWardner** [comment]
I guess it's probably not a pwntools issue 

### Also Breaks
```python
import unittest
import sys
import termios
import fcntl

class TestExercise(unittest.TestCase):
    def test_remote(self):
        fd = sys.stdout
        hw = fcntl.ioctl(fd.fileno(), termios.TIOCGWINSZ, '1234')
    
if __name__ == '__main__':
    unittest.main()
```

**Comment by Arusekk** [comment]
The correct workaround would be to set PWNLIB_NOTERM=1 when testing (I assume that 'Run' works fine). Probably fixed by #1659. I am not using vscode (because it falsely claims to be open-source, while being both closed-source [just read its EULA], and [EEE](https://en.wikipedia.org/wiki/Embrace,_extend,_and_extinguish)), so I cannot tell if this is really fixed. Feel free to reopen.
